src/Sulu/Bundle/MediaBundle/Resources/public/js/components/area-selection/demo.html

```diff
@@ -0,0 +1,81 @@
+<!doctype html>
```

Comment:
I don't know if this file is such a good idea... It will be available in every Sulu installation, without any additional value. Also I think that this file might get easily outdated and stop working correctly anyway.

Is it really necessary to have this file?